x-pack/plugins/security_solution/public/common/hooks/use_upselling.test.tsx

```diff
@@ -47,4 +47,14 @@ describe('use_upselling', () => {
     const { result } = renderHook(() => useUpsellingPage(SecurityPageName.hosts));
     expect(result.current).toBe(TestComponent);
   });
+
+  test('useUpsellingMessage returns pages', () => {
+    const testMessage = 'test message';
+    mockUpselling.registerMessages({
+      investigation_guide: testMessage,
+    });
+
+    const { result } = renderHook(() => useUpsellingMessage('investigation_guide'));
+    expect(result.current).toBe(testMessage);
+  });
```

Comment:
Since, `upSellingMessageId` can be a `string | null`, consider adding test of null `upsellingMessageId` as well.